mteb/tasks/BitextMining/multilingual/IN22ConvBitextMining.py

```diff
@@ -101,8 +101,11 @@ class IN22ConvBitextMining(AbsTaskBitextMining, MultilingualTask):
 note={}
 }""",
         descriptive_stats={
-            "avg_character_length": {
-                "test": {
+            "test": {
+                "average_sentence1_length": 54.32948595562498,
+                "average_sentence2_length": 54.32948595562498,
```

Comment:
This is exactly the same?

Reply:
This is a parallel dataset, so first one language counts as sententece1 and other time as sentence2. At total there all texts in both "sentences".